File: app/src/main/java/ca/coffeeshopstudio/meksheets/data/legacy/Pilot.java
package ca.coffeeshopstudio.meksheets.data.legacy;

public class Pilot {
    private int piloting = 5;
    private int gunnery = 4;
    private int hits;

    public int getPiloting() {
        return piloting;
    }

    public void setPiloting(int piloting) {
        this.piloting = piloting;
    }

    public int getGunnery() {
        return gunnery;
    }

    public void setGunnery(int gunnery) {
        this.gunnery = gunnery;
    }

    public int getHits() {
        return hits;
    }

    public void setHits(int hits) {
        this.hits = hits;
    }
}

File: app/src/main/java/ca/coffeeshopstudio/meksheets/data/Pilot.kt
package ca.coffeeshopstudio.meksheets.data

import androidx.compose.ui.graphics.Color

data class Pilot(var name: String = "Pilot",
                 var piloting: Int = 5,
                 var gunnery: Int = 4,
                 var color: ULong = Color.Black.value,
                 var hits: Int = 0)